doc/model.rst

```diff
@@ -0,0 +1,683 @@
+  the model, with customizable tags corresponding to values users intend to
+  modify. The default tag is a semicolon (e.g., THERMO = ;10;).
+
+In the `Example` subsection, we provide an example illustrating using the value `to_configure`
+within ``attach_generator_files()``.
+
+-------
+Example
+-------
+This example provides a demonstration of how to instruct SmartSim to attach a file
+containing parameters for use within the Model using ``Model.attach_generator_files()``
+and specifying `to_configure`.
+
+We have a text file named `configure_inputs.txt`. Within the text, is the parameter, `THERMO`,
+that is assigned a single value:
+
+.. code-block:: txt
+
+   THERMO = ;10;
+
+The parameter `THERMO` is used within our application script and we would like to
+instruct SmartSim to assign the parameter value upon Model execution.
+
+To encapsulate our simulation using a Model, we must create an Experiment instance
+to gain access to the Experiment helper function that creates the Model.
+Begin by importing the Experiment module and SmartSim log module.
+Initialize an Experiment:
+
+.. code-block:: python
+
+    from smartsim import Experiment
+    from smartsim.log import get_logger
+
+    logger = get_logger("Experiment Log")
+    # Initialize the Experiment
+    exp = Experiment("getting-started", launcher="auto")
+
+Models require run settings. Create a simple ``RunSettings`` object, specifying
+the path to our application script and the executable to run the script:
+
+.. code-block:: python
+
+    # Initialize a RunSettings object
+    model_settings = exp.create_run_settings(exe="python", exe_args="/path/to/application.py")
+
+Next, initialize a ``Model`` object with ``Experiment.create_model()``
+and pass in the `model_settings` instance:
+
+.. code-block:: python
+
+    # Initialize a Model object
+    model = exp.create_model("model", model_settings)
+
+We now have a ``Model`` instance named `model`. Attach the above text file
+to the Model for use at entity runtime. To do so, we use the
+``Model.attach_generator_files()`` function and specify the `to_configure`
+parameter with the path to the text file, `configure_params.txt`:
+
+.. code-block:: python
+
+    # Attach the file to the Model instance
+    model.attach_generator_files(to_configure="path/to/configure_params.txt")
+
+When we launch the Model, using ``Experiment.start()``, the file, `configure_inputs.txt`,
+will be in the same file path as the application and SmartSim will read and assign
+Model parameters from the text file.
+
+=====================
+ML Models and Scripts
+=====================
+--------
+Overview
+--------
+SmartSim supports sending TorchScript functions, scripts, and
+TensorFlow (TF), TensorFlow Lite (TF-lite), PyTorch (PT), or ONNX models to the database at runtime
+prior to the execution of a Model for use within the workload.
+The ``Model API`` provides a subset of helper functions that support
+these capabilities:
+
```

Comment:
- Consider separating the description of the capability from the bulleted list of supported runtimes to improve overview readability.

- Consider not saying "provides a subset of helper functions" unless some other subset is specified/linked that identifies the whole capability (iow, consider just saying "provides a set of helper functions"). 

- Consider "These capabilities are exposed in the `Model` API: <bullet list follows>"

Reply:
Potentially, add a section following overview:

--------
Runtimes
--------
- TensorFlow (TF)
- PyTorch (PT)
- ...

so we don't have to repeat the same info in AI Models intro